requirements.txt

```diff
@@ -8,8 +8,8 @@ billiard==3.5.0.3
 bleach==3.1.0
 blinker==1.4
 boto==2.46.1
-boto3==1.4.4
-botocore==1.5.53
+boto3==1.7.47
+botocore==1.10.47
```

Comment:
I had to upgrade these as the `iter_chunk()` method was not in our existing version.